src/main/java/microsoft/exchange/webservices/data/util/DateTimeUtils.java

```diff
@@ -114,4 +115,577 @@ private static Date parseInternal(String value, boolean dateOnly) {
     };
   }
 
+  /**
```

Comment:
I figured it was such a big block it might be more useful (and less clutter) in utils in case someone else needs it. I went back and forth on it, no strong preference can easily move it back